### PR TITLE
Add DBfuture class

### PR DESCRIPTION
**Pull request by hari387**
Noticed that the following pattern was commonly used:
```
try {
  future.get();
} catch (InterruptedException e) {
  System.out.println("Attempt to add recipe raised exception: " + e);
} catch (ExecutionException e) {
  System.out.println("Attempt to add recipe raised exception: " + e);
}
```
The changes refactor this into its own class, `DBFuture`, which can then be used instead.

### CODE REVIEW

**Comment by hari387** [comment]
Also, I used `e.printStackTrace` which is better compared to just `"Attempt to add recipe raised exception`.

File: src/main/java/com/google/sps/data/DBFuture.java [hunk 1]
```diff
@@ -0,0 +1,17 @@
+package com.google.sps.meltingpot.data;
+
+import com.google.api.core.ApiFuture;
+import java.util.concurrent.ExecutionException;
+
+public class DBFuture {
```
Comment (hanpeiz) [inline src/main/java/com/google/sps/data/DBFuture.java:6]: I think this should be called DBUtil. The method can be called getFuture or something. I'd expect we have more util functions for DB at some point.
Actually this is also a good place to find all the constants for your collection names and stuff.

Comment (nonsensicle) [reply src/main/java/com/google/sps/data/DBFuture.java:6]: Agree, might want to just collapse DBRefs and DBFuture

Comment (hari387) [reply src/main/java/com/google/sps/data/DBFuture.java]: Makes sense, done in 3e85a01

**Pull request merged**